modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/UpdateCommand.java

```diff
@@ -40,14 +44,21 @@ public class UpdateCommand extends PartitionCommand {
     /**
      * Creates a new instance of UpdateCommand with the given row to be updated. The {@code rowId} should not be {@code null}.
      *
+     * @param commitTablePartitionId Committed table partition id.
```

Comment:
Here and everywhere let's be consistent in naming commit/committed.

Reply:
I always use commitReplicationGroupId as a name of variable.

Reply:
Nope, sometimes it's `TablePartitionId committedPartition` and besides of that there are

- javadocs, e.g.`commitTablePartitionId Committed` 
- method names  `TablePartitionId committedPartition();`